spec/system/backend/solidus_stripe/orders/payments_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require 'solidus_stripe_spec_helper'
+
+RSpec.describe 'SolidusStripe Orders Payments', :js do
+  include SolidusStripe::BackendTestHelper
+  stub_authorization!
+
+  context 'with successful payment operations' do
+    it 'navigates to the payments page' do
+      payment = create_authorized_payment
+      visit_payments_page
+
+      expects_page_to_display_payment(payment)
+    end
+
+    it 'displays log entries for a payment' do
+      payment = create(:stripe_payment, :captured, order: order, payment_method: payment_method)
+
+      visit_payments_page
+
+      visit_payment_page(payment)
```

Comment:
Another nit: if we `visit` pages without any expectation or check we could just go to the payment page. Alternatively it could be:

```rb
visit "/admin/orders/#{order.number}/payments"
click_on payment.number
```

Ensuring we navigate the site with "clicks" after the initial visit.

Wither way is fine by me 🙌